package/Linux/gateway/template/rules

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/make -f
+
+ifeq ($(DEB_HOST_ARCH),amd64)
+ARCH = x86_64
+else ifeq ($(DEB_HOST_ARCH),arm64)
+ARCH = aarch64
+else
+$(error Unsupported architecture: $(DEB_HOST_ARCH))
+endif
```

Comment:
note: libxmf is an optional dependency, so we could still allow packaging to happen, but without libxmf.so. We’re not going to deliver artifacts ourselves, but it’s fine to still allow the packaging on other platforms if someone wants to do it.